airbyte-integrations/connector-templates/destination-python/secrets/config.json.hbs

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Does it have to be hbs, or can it just be json file? No customization in here.

Reply:
Honestly I think you can delete the file for now. Yes, we want credentials ideally, but no, destination acceptance tests are really not yet positioned to have good end to end tests. /cc @aaronsteers